charts/incubator/homebridge/Chart.yaml

```diff
@@ -23,7 +23,7 @@ sources:
   - https://homebridge.io/
   - https://github.com/oznu/docker-homebridge
 type: application
-version: 3.0.8
+version: 4.0.0
```

Comment:
Keep it minor unless you know of breaking changes.
Don't overzealusly flag as breaking